createdisk-library.sh

```diff
@@ -206,6 +206,25 @@ function prepare_hyperV() {
             echo 'CONST{virt}=="microsoft", RUN{builtin}+="kmod load hv_sock"' > /etc/udev/rules.d/90-crc-vsock.rules
 EOF
 }
+function prepare_qemu_guest_agent() {
+    local vm_ip=$1
+
+    install_additional_packages ${vm_ip} qemu-guest-agent
+
+    # f36 default selinux policy blocks usage of qemu-guest-agent over vsock
+    # checkpolicy
+    /usr/bin/checkmodule -M -m -o qemuga-vsock.mod qemuga-vsock.te
+    # policycoreutils
+    /usr/bin/semodule_package -o qemuga-vsock.pp -m qemuga-vsock.mod
+
+    ${SCP} qemuga-vsock.pp core@${vm_ip}:
+    ${SSH} core@${vm_ip} 'sudo semodule -i qemuga-vsock.pp && rm qemuga-vsock.pp'
+    ${SCP} qemu-guest-agent.service core@${vm_ip}:
+    ${SSH} core@${vm_ip} 'sudo mv -Z qemu-guest-agent.service /etc/systemd/system/'
+    # ${SSH} core@${vm_ip} 'sudo restorecon /etc/systemd/system/qemu-guest-agent.service'
```

Comment:
Why is this in here?

Reply:
Left-over from the initial commit :-/ https://github.com/crc-org/snc/commit/78db482e068668caed67caf2280688090114555e#diff-95795afe77d69420567b1e1e44ff75e9dd179f9caaaa51eefa0581e1fc1067f0R183